pennylane/qchem/tapering.py

```diff
@@ -293,44 +293,39 @@ def _taper_pauli_sentence(ps_h, generators, paulixops, paulix_sector):
     for ps in _split_pauli_sentence(ps_h, max_size=PAULI_SENTENCE_MEMORY_SPLITTING_SIZE):
         ts_ps += ps_u @ ps @ ps_u  # helps restrict the peak memory usage for u @ h @ u
 
-    wireset = ps_u.wires + ps_h.wires
+    wireset = ps_h.wires + ps_u.wires
     wiremap = dict(zip(list(wireset.toset()), range(len(wireset) + 1)))
     paulix_wires = [x.wires[0] for x in paulixops]
 
-    o = []
-    val = np.ones(len(ts_ps))
-
-    wires_tap = [i for i in ts_ps.wires if i not in paulix_wires]
-    wiremap_tap = dict(zip(wires_tap, range(len(wires_tap) + 1)))
-
-    for i, pw_coeff in enumerate(ts_ps.items()):
-        pw, _ = pw_coeff
+    wires_tap = [i for i in wiremap.keys() if i not in paulix_wires]
+    wires_ord = list(range(len(wires_tap)))
+    wiremap_tap = dict(zip(wires_tap, wires_ord))
```

Comment:
Minor: could do this in one for loop.

Reply:
I use two of these variables later on in the code, so would prefer to leave it this way.

Reply:
```suggestion
    wires_tap, wires_ord, wiremap_tap, length = [], [], {}, 0
    for i in wiremap.keys():
        if i not in paulix_wires:
            wires_tap.append(i)
            wires_ord.append(length)
            length += 1
            wiremap_tap[i] = length
```

Reply:
overkill for these variables.

Reply:
Let's review this some more. I think that you don't need these variables.

When you do
`wiremap_tap[wire]: pw[wire] for wire in wires_tap`, what is the expected value?

From what I can tell, you will always get a dictionary that has keys ascending from 0 to N. Do we _need_ this dictionary?